upgrade/upgrade-scripts/upgrade

```diff
@@ -23,6 +23,8 @@ IMAGE_PATH=$(get_image_path)
 IMAGE_VERSION=$(get_image_version)
 [[ -n "$IMAGE_VERSION" ]] || die "failed to determine image version"
 
+HOTFIX_PATH="/etc/hotfix"
```

Comment:
If you want to use a variable for this path, rather than inline it below where it's used, I'd suggest we move it into `common.sh`, in case it's ever needed in another script (at some later time).. that'll be more consistent with other similar variables/paths too.. e.g. `UPDATE_DIR`, `LOG_DIRECTORY`, etc.

Reply:
Sure

Reply:
Done.